api/s2n.h

```diff
@@ -67,6 +67,13 @@ typedef enum {
     S2N_EXTENSION_CERTIFICATE_TRANSPARENCY = 18,
 } s2n_tls_extension_type;
 
+typedef enum {
+    S2N_TLS_MAX_FRAG_LEN_512 = 1,
+    S2N_TLS_MAX_FRAG_LEN_1024 = 2,
+    S2N_TLS_MAX_FRAG_LEN_2048 = 3,
+    S2N_TLS_MAX_FRAG_LEN_4096 = 4,
+} max_frag_len;
```

Comment:
Can you rename this to `s2n_max_frag_len` for consistency? Every other enum we export as part of the public API is prefixed with `s2n`.